vsintegration/src/FSharp.ProjectSystem.PropertyPages/FSharp.PropertiesPages.vbproj

```diff
@@ -92,46 +92,24 @@
     </Compile>
     <Compile Include="Package\Constants.vb" />
     <Compile Include="Package\InternalException.vb" />
-    <Compile Include="PropertyPages\ApplicationPropPage.vb">
-      <SubType>UserControl</SubType>
-    </Compile>
-    <Compile Include="PropertyPages\ApplicationPropPageBase.vb">
-      <SubType>UserControl</SubType>
-    </Compile>
-    <Compile Include="PropertyPages\BuildEventCommandLineDialog.vb">
-      <SubType>Form</SubType>
-    </Compile>
+    <Compile Include="PropertyPages\ApplicationPropPage.vb" />
```

Comment:
I think you didn’t intend on committing these changes

Reply:
Not these again...

Reply:
yeegaw